python/dolfinx/fem/bcs.py

```diff
@@ -132,7 +132,23 @@ def function_space(self) -> dolfinx.fem.FunctionSpace:
         """The function space on which the boundary condition is defined"""
         return self._cpp_object.function_space
 
-    def dof_indices(self) -> tuple[np.ndarray, int]:
+    def set(
```

Comment:
Would be good to have the same docstring with stuff about x_0 etc. here.

Reply:
Done.